src/Containers/Lobby/styles.ts

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+
+import palette from 'palette';
+import Button from 'Components/Button';
+
+export const PlayerList = styled('div')`
+  position: static;
+  background-color: ${palette.cream};
+  opacity: 70%;
+  border: 1px solid ${palette.portTudorHotel};
+  border-radius: 10px;
+  width: 100%;
+  height: 100%;
+  min-height: 200px;
+  min-width: 100px;
```

Comment:
Check if you can get rid of them

Reply:
I can't, beacuse I want player list to be static.